packages/workspace/tests/features/workspace.feature

```diff
@@ -116,11 +150,20 @@ Scenario: Call registerService method with a service that doesnt's exist in conf
     And   I call registerService method with service C
     Then  I expect to receive an error
 
+#4.4
 Scenario: Call registerService method with invalid reference rejects servicePromise in ServicesMap
     Given WorkspaceFactory instance with createWorkspace method
     And   Configuration for token 123 that includes service A and B and components 1 and 2
     And   Service A and service B include a bootstrap that calls registerService
     When  I run createWorkspace with token 123 and Workspace is created
     And   I call services method
     And   I call workspace registerService method with invalid reference
-    Then  I expect servicePromise to be rejected with an error
\ No newline at end of file
+    Then  I expect servicePromise to be rejected with an error
+
+#4.5
+Scenario: If scalecube error happens while registering a service, the promise for this service should be rejected with an error
+    Given WorkspaceFactory instance with createWorkspace method
+    And  Configuration for token 123 that includes service A, B and D and components 1 and 2
+    When I run createWorkspace method with token 123
+    And  A scalecube error occurs while registering service D
+    Then I expect to receive an error with the name of the corresponding service
```

Comment:
this error will be received, when you will try to get the data from the promise from service D

Reply:
ok, I made the change